lumigator/jobs/inference/model_clients/huggingface_clients.py

```diff
@@ -161,6 +162,7 @@ def __init__(self, config: InferenceJobConfig):
         # HF pipeline task overwritten to 'text-generation' since these causalLMs are not task-specific models
         pipeline_config = config.hf_pipeline.model_dump()
         pipeline_config["task"] = TaskType.TEXT_GENERATION
+        pipeline_config["use_auth_token"] = config.api_key,
```

Comment:
```suggestion
        pipeline_config["token"] = config.api_key,
```

`use_auth_token` appears to be deprecated:

https://github.com/huggingface/transformers/blob/6bc0fbcfa7acb6ac4937e7456a76c2f7975fefec/src/transformers/pipelines/__init__.py#L762